hydroshare_jupyter_sync/fs_event_handler.py

```diff
@@ -0,0 +1,92 @@
+from watchdog.events import (
+    FileSystemEventHandler,
+    PatternMatchingEventHandler,
+    FileCreatedEvent,
+    FileModifiedEvent,
+    FileDeletedEvent,
+    FileMovedEvent,
+    FileClosedEvent,
+)
+
+from .fs_events import Events
+from .lib.filesystem.fs_resource_map import LocalFSResourceMap
+from .lib.events.event_broker import EventBroker
+
+from functools import wraps
+import logging
+
+# module level log
+logger = logging.getLogger(__name__)
+
+
+def log_event(fn):
+    @wraps(fn)
+    def wrapper(self, event):
+        logger.info(event)
```

Comment:
The usage on this feels more like debug

Reply:
That's a fair point. Ill change it to `debug` so it is a little more appropriate.